src/sensai/evaluation/eval_stats/eval_stats_classification.py

```diff
@@ -83,7 +83,7 @@ def __init__(self, y_predicted: PredictionArray = None,
         :param y_predictedClassProbabilities: a data frame whose columns are the class labels and whose values are probabilities
         :param labels: the list of class labels
         :param metrics: the metrics to compute for evaluation; if None, use default metrics
-        :param additionalMetrics: the metrics to additionally compute
+        :param additionalMetrics: the metrics to additionally compute. This should only be provided if metrics is None
```

Comment:
The addition is not required as there is no such restriction.

Reply:
But it is a valid recommendation for the user. It does not make sense to provide additional metrics when the default ones have been overrode, right?

Reply:
It could make sense if the user has his own defaults and only in same cases needs additional metrics. In that case, the user's default array would be altered.

Reply:
I removed the addition